src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -400,38 +443,31 @@ export function openFacilityPageV2(page, uiSchema, schema) {
       }
 
       const eligibilityChecks =
-        newAppointment.eligibility[`${locationId}_${typeOfCareId}`] || null;
+        newAppointment.eligibility[`${facilityId}_${typeOfCareId}`] || null;
 
       if (eligibilityDataNeeded && !eligibilityChecks) {
-        const parentId = getParentOfLocation(parentFacilities, locations[0]).id;
+        const selectedFacility = typeOfCareFacilities[0];
+        if (!parentId) {
```

Comment:
Similar to above, if you need the siteId, you can get the first 3 digits of the facility id.

We've avoided that in the past due to the parent and site distinction, but as far as I know none of the apis outside of the direct_scheduling_facilities (which we're now skipping) care about the parent over the site id.